source/Octopus.Client/Model/StepsResource.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Octopus.Client.Extensibility;
+
+namespace Octopus.Client.Model
+{
+    public class StepsResource : Resource, IHaveSpaceResource
```

Comment:
This inherits `DeploymentProcessBaseResource` right?

Reply:
No, this was from a bad cherry-pick, so this was the old class name before we renamed everything to OpsSteps ... removing this now.